src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -0,0 +1,287 @@
+     *
+     * Default: 1.0
+     */
+    private String bundleVersion;
+
+    /**
+     * A user-visible string for the release or version number of the bundle. A
+     * string composed of one to three period-separated integers, containing
+     * numeric characters (0-9) and periods only.
+     *
+     * Default: 1.0
+     */
+    private String bundleShortVersion;
+
+    /**
+     * String that identifies a valid certificate that will be used for iOS development
+     * or iOS distribution.
+     *
+     * Default: null. When not provided, Substrate will be selected from all the valid identities found
+     * installed on the machine from any of these types:
+     *
+     *      iPhone Developer|Apple Development|iOS Development|iPhone Distribution
+     *
+     * and that were used by the provisioning profile.
+     */
+    private String providedSigningIdentity;
+
+    /**
+     * String with the name of the provisioning profile created for iOS development or
+     * distribution of the given app.
+     *
+     * Default: null. When not provided, Substrate will try to find a valid installed
+     * provisioning profile that can be used to sign the app, including wildcards.
+     */
+    private String providedProvisioningProfile;
+
+    /**
+     * Boolean that can be used to skip signing iOS apps. This will prevent any
+     * deployment, but can be useful to run tests without an actual device
+     */
+    private boolean skipSigning;
+
+    /**
+     * A string with a valid name of an iOS simulator device
+     */
+    private String simulatorDevice;
+
+    // Android
+
+    /**
+     * A user-visible short name for the app
+     *
+     * Default: if not set, $appName will be used.
+     */
+    private String appLabel;
+
+    /**
+     * A positive integer used as an internal version number
+     *
+     * Default: 1
+     */
+    private String versionCode;
+
+    /**
+     * A string used as the version number shown to users, like
+     * <major>.<minor>.<point>
+     *
+     * Default: 1.0
+     */
+    private String versionName;
+
+    /**
+     * A string with the path to a keystore file that can be used to sign
+     * the Android apk.
+     *
+     * Default: null. If not set, Substrate creates and uses a debug keystore.
+     */
+    private String providedKeyStorePath;
+
+    /**
+     * A string with the password of the provide keystore file.
+     *
+     * Default: null. If not set, Substrate creates and uses a debug keystore.
+     */
+    private String providedKeyStorePassword;
+
+    /**
+     * A string with an identifying name for the key
+     *
+     * Default: null. If not set, Substrate creates and uses a debug keystore.
+     */
+    private String providedKeyAlias;
+
+    /**
+     * A string with a password for the key
+     *
+     * Default: null. If not set, Substrate creates and uses a debug keystore.
+     */
+    private String providedKeyAliasPassword;
+
+    public ReleaseConfiguration() {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5455854)